og.module

```diff
@@ -150,9 +150,8 @@ function og_entity_create_access(AccountInterface $account, array $context, $bun
     }
 
     $handler = Og::getSelectionHandler($field_definition);
-
-    if ($handler->getReferenceableEntities()) {
-      return AccessResult::neutral();
+    if ($handler->countReferenceableEntities()) {
+      return AccessResult::allowed();
```

Comment:
If the user gets access based on OG, and doesn't have site wide permission to create content, we need to set the audience field to be required.

Things become more complicated when we might have multiple audience fields :/